coalib/coala_main.py

```diff
@@ -127,7 +127,7 @@ def run_coala(console_printer=None,
             cache.write()
 
         if did_nothing:
-            nothing_done(log_printer)
+            nothing_done()
```

Comment:
hmmm, should we still allow them to specify a logging object for log_printer? (or am I misreading this snippet?)